Wordle.UI/ConsoleUI.cs

```diff
@@ -72,10 +70,11 @@ public void UpdateBoard(WordScore[] words)
     }
     private void RenderRow(WordScore word)
     {
-        foreach (LetterScore letter in word.LetterScores)
+        var sortedLS = word.LetterScores.OrderBy(ls => ls.Id);
```

Comment:
nit: 
`sortedLS` --> `sortedLetterScore`
`ls` --> `letterScore`

Open to @MarieRobinson's or Ian's thoughts about variable naming but I think full words vs abbreviations is preferred!